### ISSUE TITLE
Connection problem between ESP3266 and alexa on amazon Echo

### ISSUE THREAD

**Issue by yannicktremblay3** [kakopappa/arduino-esp8266-alexa-multiple-wemo-switch#68]
Hello,

I bought a Amazon echo last week and I did some test with my ESP3266 to eventually make a project... Last week all is worked but this week my amazon echo can't find my ESP3266 on my network, I did no change since last week... I tried to reset my network, reset my echo, put some program that worked lasw week on my esp but it always cant find my esp... I tried an other esp3266 module and had the same problem. Someone can help me?

 

Thanks

**Comment by kakopappa** [comment]
Use https://Sinric.com

**Comment by Ynot1** [comment]
You are only trying to connect a single esp device right? - that code doesn’t scale up without some minor tweaks to make each device unique.

Otherwise, Delete the device from your amazon account and try discovery again.



Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by yannicktremblay3** [comment]
kakopappa, I tried sinric but it can't detec my device anyway.
Ynot1, I deleted my ESP to my account I can't see it on my connected device but when I try to detect it's always the same problem.

**Comment by yannicktremblay3** [comment]
I finally find the problem, I changed my wi-fi rooter  and it solve the problem. But I dont understand why my last rooter blocked the connection... it have 3 internal antenna but I also tried to connect all of my device on the same 2.4ghz antenna (because we can't connect ESP to 5 ghz) and the problem was the same. I thinked that if all of my device is on the same network and same antenna it could resolve the problem but no... someone have some explain?
